bin/pipeline.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+COLLECTION=shared/ci_hsc_output
+
+pipetask -d "patch.patch = 69" -j "$1" -b "$CI_HSC_GEN3_DIR"/butler.yaml -p lsst.meas.base -p lsst.ip.isr -p \
+lsst.pipe.tasks -i calib,shared/ci_hsc -o "$COLLECTION" run \
+--register-dataset-types \
+-t isrTask.IsrTask:isr -C isr:"$CI_HSC_GEN3_DIR"/configs/isr.py \
+-t characterizeImage.CharacterizeImageTask:cit -C cit:"$CI_HSC_GEN3_DIR"/configs/charImage.py \
+-t calibrate.CalibrateTask:ct -C ct:"$CI_HSC_GEN3_DIR"/configs/calibrate.py \
+-t makeCoaddTempExp.MakeWarpTask:mwt -C mwt:"$CI_HSC_GEN3_DIR"/configs/makeWarp.py \
+-t assembleCoadd.CompareWarpAssembleCoaddTask:cwact -C cwact:"$CI_HSC_GEN3_DIR"/configs/compareWarpAssembleCoadd.py \
+-t multiBand.DetectCoaddSourcesTask \
+-t mergeDetections.MergeDetectionsTask:mdt -C mdt:"$CI_HSC_GEN3_DIR"/configs/mergeCoaddDetections.py \
+-t deblendCoaddSourcesPipeline.DeblendCoaddSourcesSingleTask \
```

Comment:
Would it make sense to add config for DetectCoaddSourcesTask and DeblendCoaddSourcesSingleTask too, just to be more future proof?